app/classifier/tasks/survey/image-flipper.jsx

```diff
@@ -49,19 +52,19 @@ class ImageFlipper extends React.Component {
                   className={`survey-task-image-flipper-pip ${index === this.state.frame ? 'active' : ''}`}
                 >
                   <input
+                    aria-label={`Example ${index + 1}`}
```

Comment:
Based on feedback in Talk, this could be more explicit about what the radio button does eg. 'View example 1'.